modules/console/pages/config/security/authorization.adoc

```diff
@@ -36,110 +145,117 @@ Redpanda Console comes with the following roles:
-[,yaml]
+The Redpanda Console service account must be a superuser principal in Redpanda. Without superuser privileges, Redpanda Console is unable to perform actions such as listing topics, retrieving cluster metrics, managing ACLs, or accessing administrative endpoints.
+
+To grant superuser permissions to the service account:
+
+[,bash]
 ----
-kafka:
-  brokers: ["broker1.example.com:9092"]
-  sasl:
-    enabled: true
-    impersonateUser: false <1>
-authentication: <2>
-  jwtSigningKey: secret-key
-  useSecureCookies: false
-  oidc:
-    enabled: true
-    issuerUrl: https://accounts.google.com
-    clientId: "example-client-id"
-    clientSecret: "example-client-secret"
-    redirectUrl: http://localhost:9090/auth/callbacks/oidc
-  basic:
-    enabled: true <2>
-authorization: <3>
-  roleBindings:
-  - roleName: admin
-    users:
-      - loginType: OIDC
-        name: john.doe@redpanda.com # Refers to the sub claim in the JWT token
-  - roleName: viewer
-    users:
-      - loginType: basic
-        name: Alice # Refers to the SASL/SCRAM username for the Kafka API or the basic auth username for the HTTP APIs.
+rpk cluster config set superusers '["console-superuser"]' \
+  -X user=<existing-superuser> -X pass=<password>
 ----
-<1> The `impersonateUser` flag is set to `false`, which means Redpanda Console does not impersonate the user who is logged in when making requests to the Kafka API.
-<2> The `authentication` block enables users to log into Redpanda Console using OIDC or basic authentication.
-<3> The `authorization` block defines the role bindings that grant permissions to users. If this user logs into Redpanda Console as either `john.doe@redpanda.com` or Alice, they will have the permissions of an Admin or Viewer, respectively.
 
-If the user is impersonated, like in the following example, the role binding is ignored, and the user's permissions are determined by the roles assigned to them in Redpanda:
+Before login, Redpanda Console authenticates the user by validating their credentials against the Kafka API. If the user does not exist in Redpanda (for example, if they have not been created or their identity is not mapped through OIDC), login fails.
+
+After login, all API communication is performed using the service account's credentials, not the end-user's. This ensures Redpanda Console can act on the user's behalf. As such, the service account must have sufficient privileges to cover the actions users are allowed to perform in the UI.
+
+=== Example: Assign the Viewer role
+
+This example grants the Viewer role to a user named `matt`. The Redpanda Console service account (`console-superuser`) is used to authenticate to all Redpanda services.
 
-.Role bindings ignored
 [,yaml]
 ----
 kafka:
   brokers: ["broker1.example.com:9092"]
   sasl:
     enabled: true
-    impersonateUser: true <1>
-authentication: <2>
-  jwtSigningKey: secret-key
-  useSecureCookies: false
+    impersonateUser: false
+    username: "console-superuser" # <1>
```

Comment:
Shouldn't the <1> be in the line above?
Description below reads: "With impersonation disabled, ..."
I know it mentions the superuser too, but the key setting here is that impersonation is disabled, no?